src/styles/blocks/card.scss

```
@@ -0,0 +1,97 @@
+@import "../utils/variables";
+
+.card {
+  font-family: Roboto, sans-serif;
+  width: 200px;
```

```suggestion
  width: 200px;
```
use variable
використай змінну